tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/hci.test_list

```diff
@@ -16,16 +16,16 @@ HCI/CIN/BV-03-C
 HCI/CIN/BV-04-C
 HCI/CIN/BV-06-C
 HCI/CIN/BV-09-C
-HCI/CM/BV-01-C
-HCI/CM/BV-02-C
-HCI/CM/BV-03-C
+HCI/CM/BV-01-C # [Handling LE Read Peer Resolvable Address Command]
+HCI/CM/BV-02-C # [Handling LE Read Local Resolvable Address Command]
+HCI/CM/BV-03-C # [Handling LE Read PHY Command]
 HCI/DDI/BI-02-C
 HCI/DDI/BV-03-C
 HCI/DDI/BV-04-C
 HCI/DSU/BV-02-C
-HCI/DSU/BV-03-C
+HCI/DSU/BV-03-C # [Reset Command received in Slave Role]
 HCI/DSU/BV-04-C
-HCI/DSU/BV-05-C
-HCI/DSU/BV-06-C
-HCI/GEV/BV-01-C
-HCI/HFC/BV-04-C
+#HCI/DSU/BV-05-C
+HCI/DSU/BV-06-C # [Reset Command received in Master Role]
+# HCI/GEV/BV-01-C
```

Comment:
Should these still be commented out?

Reply:
They seem to fail on the refactored control procedures, and require further investigation